pkg/services/services.go

```diff
@@ -178,7 +178,10 @@ func getKubeletServiceConfiguration(argsFromIginition map[string]string, debug b
 		preScripts = append(preScripts, hostnameOverridePowershellVar)
 	}
 
-	kubeletServiceCmd := windows.KubeletPath
+	kubeLogCmd := fmt.Sprintf("%s -log-file=%s -redirect-stderr=false ",
+		windows.KubeLogRunnerPath, windows.KubeletLog)
+
+	kubeletServiceCmd := kubeLogCmd + windows.KubeletPath
```

Comment:
Please add windows.KubeletPath as part of the Sprintf, no reason for it to be separate

Reply:
Does this need to be done for kube-proxy as well?

Reply:
yes it needs to be, I am contemplating on whether to add it in this PR or add it as part of kube-proxy bump when the new sdn branch is created. [kube-log-runner](https://github.com/kubernetes/kubernetes/tree/62d9f8ba80f4cc660a88dcc34f56c5c6f7df17ea/staging/src/k8s.io/component-base/logs/kube-log-runner) only has the functionality to direct the logs to a file. For kube-proxy, we have been redirecting to a log-dir for a long time now. Is it acceptable to move them only to info and error files?

Reply:
Would doing this for kube-proxy have effect on how we collect log files (e.g. must-gather changes)?

Reply:
If `kube-proxy` does not require this change we can leave it as is.

Reply:
@aravindhp It will once we update to the 1.26 SDN branch. 
@saifshaikh48 yes we will need to update the file info [here](https://github.com/openshift/must-gather/blob/3860800ff98445620e40d474a9f6dd2f36027fca/collection-scripts/gather_windows_node_logs#L6)